plugins/osdn/factory/factory.go

```diff
@@ -23,3 +23,10 @@ func NewPlugin(pluginType string, osClient *osclient.Client, kClient *kclient.Cl
 
 	return nil, nil, nil
 }
+
+func IsMultitenantNetworkPlugin(pluginType string) bool {
+	if strings.ToLower(pluginType) == ovs.MultiTenantPluginName() {
```

Comment:
This part is going to be fun when flannel bits land, since those need the VNID stuff too, but I'll just deal with that when things settle down.